recipes/pyode/LICENSE-BSD.TXT

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
This file isn't needed as it's included in the source tarball and the `licence_file` field looks there automatically. It's also preferable as it ensures the file stays up to date if the licence ever changes.

Reply:
ok, removing